src/services/motion/types.ts

```diff
@@ -67,12 +67,13 @@ export const encodeConstraints = (obj: Constraints): pb.Constraints => {
 };
 
 export const encodeMotionConfiguration = (
-  obj: MotionConfiguration
+  obj: MotionConfiguration = {}
```

Comment:
@micheal-parks since I added this, do you think I should remove 
```
    if (motionConfig) {
      request.setMotionConfiguration(encodeMotionConfiguration(motionConfig));
    }
```
from `src/services/motion/client.ts`?

Reply:
I think it still makes sense to not call this function if no config is provided